meta-mender-toradex-nxp/classes/image_type_mender_tezi.bbclass

```diff
@@ -0,0 +1,195 @@
+# This class implements a Mender-enabled Toradex Easy Installer image
+# Based largely on meta-toradex-bsp-common/classes/image_type_tezi.bbclass
+
+# Make sure the full Mender multi-partition image is available
+IMAGE_TYPEDEP:mender_tezi:append = "${@bb.utils.contains('IMAGE_FSTYPES', 'sdimg', ' sdimg.gz', '', d)} \
+                                       ${@bb.utils.contains('IMAGE_FSTYPES', 'ubimg', ' ubimg.gz', '', d)} \
+                                       ${@bb.utils.contains('IMAGE_FSTYPES', 'gptimg', ' gptimg.gz', '', d)} \
+                                       ${@bb.utils.contains('IMAGE_FSTYPES', 'uefiimg', ' uefiimg.gz', '', d)} \
+                                       ${@bb.utils.contains('IMAGE_FSTYPES', 'biosimg', ' biosimg.gz', '', d)}"
+
+# Figure out what type of image we are basing this on
+FULL_IMAGE_SUFFIX = ""
+FULL_IMAGE_SUFFIX_mender-image-sd = "sdimg"
+FULL_IMAGE_SUFFIX_mender-image-ubi = "ubimg"
+FULL_IMAGE_SUFFIX_mender-image-uefi = "uefiimg"
+FULL_IMAGE_SUFFIX_mender-image-bios = "biosimg"
+FULL_IMAGE_SUFFIX_mender-image-gpt = "gptimg"
+
+TEZI_AUTO_INSTALL ??= "false"
+TEZI_CONFIG_FORMAT ??= "2"
+
+# Do not include these image types:
+IMAGE_FSTYPES:remove = "${SOC_DEFAULT_IMAGE_FSTYPES} tar.xz ${FULL_IMAGE_SUFFIX}.bz2"
+
+WKS_FILE_DEPENDS:append = " mender-tezi-metadata "
+
+do_image_mender_tezi[recrdeptask] += "do_deploy"
+RM_WORK_EXCLUDE += "${PN}"
+
+def rootfs_mender_tezi_emmc(d):
+    from collections import OrderedDict
+    offset_bootrom = d.getVar('OFFSET_BOOTROM_PAYLOAD')
+    offset_spl = d.getVar('OFFSET_SPL_PAYLOAD')
+    imagename = d.getVar('IMAGE_LINK_NAME')
+    image_suffix = d.getVar('FULL_IMAGE_SUFFIX')
+    storage_device = os.path.basename(d.getVar('MENDER_STORAGE_DEVICE'))
+
+    bootpart_rawfiles = []
+
+    if offset_spl:
+        bootpart_rawfiles.append(
+              {
+                "filename": d.getVar('SPL_BINARY'),
+                "dd_options": "seek=" + offset_bootrom
+              })
+    bootpart_rawfiles.append(
+              {
+                "filename": d.getVar('UBOOT_BINARY_TEZI_EMMC'),
+                "dd_options": "seek=" + (offset_spl if offset_spl else offset_bootrom)
+              })
+
+    return [
+        OrderedDict({
+          "name": storage_device,
+          "table_type": "gpt",
+              "content": {
+                  "rawfiles": [
+                      {
+                          "dd_options": "bs=8M",
+<<<<<<< HEAD
```

Comment:
This looks like a broken merge, can you please check?

Reply:
Again, this was present in the original commit that I reverted. These artifacts were introduced in earlier commits that I am fairly certain never got tested.

I fixed these issues in a subsequent atomic commit (440a192d7dd104f452fc3023b286827d4f156418).